enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -145,6 +145,13 @@ class SubsidyAccessPolicyCRUDViewset(PermissionRequiredMixin, viewsets.ModelView
     # impossible for learners to create policies.
     permission_required = REQUESTS_ADMIN_LEARNER_ACCESS_PERMISSION
 
+    def __init__(self, *args, **kwargs):
+        self.extra_context = {}
+        super().__init__(*args, **kwargs)
+
+    def set_idempotent_policy(self, idempotent):
+        self.extra_context['idempotent'] = idempotent
```

Comment:
The function name seems ambiguous to me.  Like, isn't the policy always idempotent, why do we have to call a function to make it idempotent?  IMO:
```suggestion
    def set_policy_created(self, created):
        self.extra_context['created'] = created
```
then, above, call it with True and False in both branches.